environs/config/config.go

```diff
@@ -370,35 +374,28 @@ func (method HarvestMode) HarvestUnknown() bool {
 	return method&HarvestUnknown != 0
 }
 
-// HasDefaultSeries defines a interface if a type has a default series or not.
-type HasDefaultSeries interface {
-	DefaultSeries() (string, bool)
-}
-
 // GetDefaultSupportedLTS returns the DefaultSupportedLTS.
```

Comment:
GetDefaultSupportedLTSBase comment block should be updated for bases rather than series